generate_small_test_ref.py

```diff
@@ -90,8 +90,6 @@ def uint(value):
     except ValueError: logger.error("Failed to retrieve %s from %s" % (str(region).strip(), invcf))
     if records is None: continue
     for record in records:
-      if not record:
```

Comment:
Yes, @roger-liu-bina it should not be empty. `not record` was not there earlier and it worked. Roger added to his pull request and later removed it.